ps_emailsubscription.php

```diff
@@ -98,6 +98,9 @@ public function install()
                     'actionCustomerAccountAdd',
                     'additionalCustomerFormFields',
                     'displayAdminCustomersForm',
+                    'registerGDPRConsent',
```

Comment:
Do you really need this hook `registerGDPRConsent`?

Reply:
Yes, if this hook is not present the module will not be registered in the official gdpr module